Tasks/ServiceFabricDeployV1/deploy.ps1

```diff
@@ -107,11 +111,13 @@ try {
 
         if (!$skipValidation)
         {
+            $global:operationId = $SF_Operations.TestApplicationPackage
```

Comment:
Should we add Get Application as operation as well

Reply:
Vinod did not add it initially. I am not sure whether he had a reason. But I agree ideally we should add.

Reply:
done

Reply:
Added a few other missing as well